kvm-install-vm

```diff
@@ -679,7 +679,6 @@ _EOF_
     # Assemble CI ISO disk parameters.
     CI_ISO_PARAMS="$(join ',' \
         ${CI_ISO} \
-        $(param device ${CI_ISO_DEVICE}) \
```

Comment:
Should we also remove the line?

    CI_ISO_DEVICE=cdrom

In the function set_defaults()

Reply:
Thanks for reviewing, you are right! Fixed that.